server/models/MemberInvitation.js

```diff
@@ -241,7 +241,7 @@ function defineModel() {
       invitation = await MemberInvitation.create({ ...memberParams, CollectiveId: collective.id }, sequelizeParams);
       invitation.collective = collective;
 
-      if ([roles.ACCOUNTANT, roles.ADMIN, roles.MEMBER].includes(this.role)) {
+      if (MEMBER_INVITATION_SUPPORTED_ROLES.includes(memberParams.role)) {
```

Comment:
🐞 `this.role` is undefined here, so it wasn't actually creating the `COLLECTIVE_CORE_MEMBER_INVITED` activity.

Reply:
Good catch, it's similar to the fix made in https://github.com/opencollective/opencollective-api/pull/7598.